### PR TITLE
FIX => Goods Details not getting created 

### PR DESCRIPTION
**Pull request by abulsayyad123**
Hi Team,
This PR is for goods not getting added on goods_details page during Booking Appointment. Also fix for ticket GCW-2528-client details not getting updated (https://jira.crossroads.org.hk/browse/GCW-2528). Please, review this PR and let me know if any change is required.

Thanks.

### CODE REVIEW

File: app/controllers/order/goods_details.js [hunk 1]
```diff
@@ -14,12 +14,16 @@ export default Ember.Controller.extend(cancelOrder, {
...
   otherDetails: "",
   sortProperties: ["id"],
-  sortedGcRequests: Ember.computed("gcRequests.[]", function() {
-    return this.get("gcRequests");
-  }),
+  sortedGcRequests: Ember.computed(
+    "order.goodcityRequests",
+    "sortProperties",
+    function() {
+      return this.get("order.goodcityRequests");
```
Comment (Darshan-Chauhan) [inline app/controllers/order/goods_details.js]: Is this sorted?? Because it doesn't look like it is

Comment (abulsayyad123) [reply app/controllers/order/goods_details.js]: Yes, it is sorted according to order of its creation.

Comment (Darshan-Chauhan) [reply app/controllers/order/goods_details.js]: Because I see it's dependent on `sortProperties` but it is not used

Comment (abulsayyad123) [reply app/controllers/order/goods_details.js]: Got the issue, thanks for clarifying. Added sortProperties in sort computed property.

File: app/models/order.js [hunk 2]
```diff
@@ -97,15 +97,15 @@ export default Model.extend({
     }
   ),
 
-  clientIdNumber: Ember.computed("beneficiary", function() {
+  clientIdNumber: Ember.computed("beneficiary.identityNumber", function() {
```
Comment (patrixr) [inline app/models/order.js]: those can probably be shortened with `Ember.computed.alias('beneficiary.identityNumber')` :D

**Pull request merged**